packages/react-dev-overlay/src/middleware-turbopack.ts

```diff
@@ -0,0 +1,143 @@
+import type { IncomingMessage, ServerResponse } from 'http'
+import type { StackFrame } from 'stacktrace-parser'
+import type { ParsedUrlQuery } from 'querystring'
+import type { OriginalStackFrameResponse } from './middleware'
+
+import fs, { constants as FS } from 'fs/promises'
+import url from 'url'
+import { codeFrameColumns } from '@babel/code-frame'
+import { launchEditor } from './internal/helpers/launchEditor'
+
+interface Project {
```

Comment:
Not sure how to import these from the `next` package since that depends on this package...